### DISCUSSION TITLE
DS75%40UCSD/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#228] [Announcements]
# DS75%40UCSD/

None

https://opencs.app/DS75%40UCSD/

**Comment by Yangxinyee** [comment]
这个项目有uu申过嘛，咋样呀？

**Reply by Stepbrotherimstuck** [reply]
有 不过听说好像今年偏好数学的多 应数专业 专gpa3.7 总3.5 没托福 gre325+3.5

**Comment by angl1n** [comment]
有没有好兄弟知道这个是统计课多还是能全选软课
